### PR TITLE
JUnit5 extension

### PR DESCRIPTION
**Pull request by jartysiewicz**

### CODE REVIEW

File: src/main/java/de/cronn/assertions/validationfile/junit5/ValidationFileSupport.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package de.cronn.assertions.validationfile.junit5;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import de.cronn.assertions.validationfile.ValidationFileAssertions;
+
+public class ValidationFileSupport {
+	private static final Map<Long, String> currentTestNames = Collections.synchronizedMap(new HashMap<>());
```
Comment (essmeyer) [inline src/main/java/de/cronn/assertions/validationfile/junit5/ValidationFileSupport.java]: Have you considered using a ConcurrentHashMap here?

Comment (jartysiewicz) [reply src/main/java/de/cronn/assertions/validationfile/junit5/ValidationFileSupport.java]: good idea

**Comment by jartysiewicz** [comment]
@essmeyer one more thing, I added one feature that I need urgently in other project. It's ability to configure base directory for validation files. Check it out https://github.com/cronn/validation-file-assertions/pull/3/commits/6054a43af720b340b15825241154eef7048a7aed

**Comment by jartysiewicz** [comment]
@essmeyer conversation was messed, probably due to the fact that I force pushed my branch. Anyway, you made claim about the name of `DirsConfig` . 

From my perspective, any name fits here as long as this piece of code is not intended to be used directly by the library user. The name of the piece of code that was predecessor was: `TestData` so with name `DirsConfig` I see enormous progress in the matter :)

**Pull request closed without merging**